openpgp.go

```diff
@@ -39,59 +37,26 @@ func fold256(x []byte) (y b256) {
 }
 
 func xor(a, b b256) (c b256) {
-	for i := 0; i<len(a); i++ {
+	for i := 0; i < len(a); i++ {
 		c[i] = a[i] ^ b[i]
 	}
 	return c
 }
 
 // Any hash function may have restricted range of output, so we XOR the
 // original every time, which could increase the range of output.
-// XXX test.
 func next256(ent b256, last b256) (next b256) {
 	h := sha256.New()
 	x := xor(ent, last)
 	h.Write(x[:])
 	s := h.Sum(nil)
 	copy(next[:], s)
-	fmt.Printf("ent %X last %X xor %X hash %X\n", ent, last, x, s)
 	return
 }
 
-func test() {
-	fmt.Println(min(1,2))
-	fmt.Println(min(2,1))
-	fmt.Println(fold256([]byte("a")))
-	fmt.Println(fold256([]byte("abc")))
-	fmt.Println(fold256([]byte("1234567890")))
-	fmt.Println(fold256([]byte("12345678901234567890123456789012")))
-	fmt.Println(fold256([]byte("123456789012345678901234567890122")))
-	fmt.Println(fold256([]byte("1234567890123456789012345678901212345678901234567890123456789012")))
-	fmt.Println(fold256([]byte("12345678901234567890123456789012123456789012345678901234567890121")))
-	{
-	fmt.Println("----------------------------------------")
-	a := b256{}
-	b := b256{}
-	fmt.Println(next256(a, b))
-	fmt.Println(next256(a, b))
-	b = next256(a, b)
-	fmt.Println(next256(a, b))
-	}
-
-	{
-	fmt.Println("----------------------------------------")
-	a := b256{}
-	a[0] = 0xFF
-	b := b256{}
-	fmt.Println(next256(a, b))
-	fmt.Println(next256(a, b))
-	b = next256(a, b)
-	fmt.Println(next256(a, b))
-	}
-}
-
 func main() {
-	test()
+	fmt.Println("WARNING: This is a test implementation. The entropy source will be replaced with custom entropy.")
```

Comment:
Is the idea to import the `gnokey` packages into this repo (for the dice rolls and card draws) or to reimplement / centralize everything here?